app/views/dashboard/_seller.html.erb

```diff
@@ -1,9 +1,6 @@
 <% props = Property.where(user_id: current_user.id) %>
-<% sold =  props.where(is_paid: true) %>
-<% sum = 0%>  
-<% sold.each do |sld| %>
-  <% sum = sum + sld.price %>
-<% end %>
+<% sold_property =  props.where(is_paid: true) %>
+
 <div class="row" style="margin-top: 130px; justify-content: center;">
```

Comment:
Remove inline styling.

Reply:
- Removed inline styles and added it to CSS